pkgs/development/ocaml-modules/gitlab/unix.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildDunePackage,
+  gitlab,
+  cmdliner,
+  cohttp,
+  cohttp-lwt-unix,
+  tls,
+  lwt,
+  stringext,
+  alcotest,
+}:
+
+buildDunePackage {
+  pname = "gitlab-unix";
+  inherit (gitlab) version src;
+
+  minimalOCamlVersion = "4.08";
+
+  postPatch = ''
+    substituteInPlace unix/dune --replace 'gitlab bytes' 'gitlab'
```

```suggestion
    substituteInPlace unix/dune --replace 'gitlab bytes' 'gitlab'
```

`replace` is deprecated.